ledger-tool/src/blockstore.rs

```diff
@@ -830,7 +827,7 @@ fn do_blockstore_process_command(
             let full_regex = Regex::new(r"slot (\d*) is full").unwrap();
 
             let log_file = PathBuf::from(value_t_or_exit!(arg_matches, "log_path", String));
-            let f = BufReader::new(File::open(log_file).unwrap());
+            let f = BufReader::new(File::open(log_file)?);
```

Comment:
So far this is the first line I've spotted that wouldn't return a `BlockstoreError`